client/my-sites/plans/plan-overview/style.scss

```diff
@@ -0,0 +1,27 @@
+@mixin plan-overview-button {
```

Comment:
After giving his some more thought, I think sharing this style using classnames is preferable to either a mixin or extend:

```scss
.plan-overview {
  .plan-feature__button,
  .plan-status__button {
    flex-shrink: 0;
    margin-left: 15px;
    min-width: 150px;
    text-align: center;

	@include breakpoint( "<480px" ) {
		margin: 15px 0 0 0;
		width: 100%;
	}
  }
}
```

Rationale:
- This is easier to grok than a mixin/extend. Defining the styles for `%plan-overview-button` in the `PlanOverview` stylesheet and including them with `@extend` elsewhere makes it difficult to know where the styles are used.
- This probably results in a smaller CSS bundle.